spec.html

```diff
@@ -35771,6 +36366,43 @@ <h1>
           </emu-alg>
         </emu-clause>
 
+        <emu-clause id="sec-maybesimplecasefolding" type="abstract operation">
+          <h1>
+            MaybeSimpleCaseFolding (
+              _A_: a CharSet,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It uses the Unicode <a href="https://www.unicode.org/reports/tr44/#Simple_Case_Folding">Simple Case Folding</a> mapping, which maps one code point to one code point, and which is abbreviated <a href="https://www.unicode.org/Public/UCD/latest/ucd/PropertyAliases.txt">scf</a>.</dd>
+          </dl>
+          <emu-alg>
+            1. If _UnicodeSets_ is *false* or _IgnoreCase_ is *false*, return _A_.
+            1. Let _S_ be an empty CharSet.
+            1. For each CharSetElement _cs_ of _A_, do
+              1. Let _t_ be an empty sequence of characters.
+              1. For each single code point _c_ in _cs_, do
+                1. Append scf(_c_) to _t_.
+              1. If _S_ does not contain _t_, then
+                1. Add _t_ to _S_.
```

Comment:
Suggested edit applied as part of https://github.com/mathiasbynens/ecma262/pull/10. Will keep this comment “unresolved” for the follow-up suggestions.

Reply:
- I don't care if we call the output S or B.
- If the output is no longer called S, then I am fine calling the element s.
- I don't care about c vs. cp.
- t works better if we have element s because t comes right after s, and they can be mnemonic for source & target. But I don't care strongly about the name.

Reply:
Variables renamed in the second commit of https://github.com/mathiasbynens/ecma262/pull/14 except keeping `t`, see above.